Build/libHttpClient.141.Win32.C/libHttpClient.141.Win32.C.vcxproj

```diff
@@ -55,7 +55,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)'=='Release'" Label="Configuration">
     <UseDebugLibraries>false</UseDebugLibraries>
-    <WholeProgramOptimization>false</WholeProgramOptimization>
+    <WholeProgramOptimization>true</WholeProgramOptimization>
     <LinkIncremental>false</LinkIncremental>
```

Comment:
can't set this to true.

see [https://docs.microsoft.com/en-us/cpp/porting/binary-compat-2015-2017?view=msvc-170](https://docs.microsoft.com/en-us/cpp/porting/binary-compat-2015-2017?view=msvc-170)